packages/components/src/DataList/components/DataListTags/DataListTags.css

```diff
@@ -7,7 +7,7 @@
 }
 
 .tagCount {
-  --overflow-bg: var(--color-white);
+  --overflow-bg: var(--data-list-item-active-color, var(--color-white));
```

Comment:
Just for my own curiousity, what does the comma-separated list of values do here? I know how it works with `background-image` but I've never seen it used for colors!

Reply:
It's CSS variable specific. It's basically a fallback when that variable doesn't exist. Which it doesn't when the parent isn't active.

Reply:
🌟 The More You Know ☄️  

Specific to this case, we should be using `color-surface` rather than the base `color-white` to define the background of the overflow, right?

Reply:
Oh you right!!

Reply:
![image](https://github.com/GetJobber/atlantis/assets/39704901/1dcad725-027e-420f-912e-8336ba2c22d3)